src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.container {
+  box-sizing: border-box;
+  padding: 50px 40px;
+  display: grid;
+  place-content: center;
+  place-items: center;
+  row-gap: 48px 46px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+
+@media (min-width: 300px) {
+  .container {
+    grid-template-columns: repeat(1, 1fr);
+  }
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to avoid using `box-sizing: border-box;` multiple times. Consider creating a common class or using inheritance to apply this property once globally.